### PR TITLE
[LOOP-4942] Use proper guidanceColors for DismissableHostingController

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4942

### CODE REVIEW

File: Loop/View Controllers/StatusTableViewController.swift [hunk 1]
```diff
@@ -1431,7 +1431,8 @@ final class StatusTableViewController: LoopChartsTableViewController {
         let hostingController = DismissibleHostingController(
             content: bolusEntryView(
                 enableManualGlucoseEntry: enableManualGlucoseEntry
-            )
+            ),
+            guidanceColors: .default
```
Comment (nhamming) [inline Loop/View Controllers/StatusTableViewController.swift]: I see that elsewhere a `DismissibleHostingController` is instantiated using 

```
DismissibleHostingController(rootView: ..., isModalInPresentation: ...)
```

which uses the .default guidance colors. Should this be used here as well?

**Review by nhamming — APPROVED** [review]
Not blocking, but maybe it should be updated for consistency.

File: Loop/View Controllers/StatusTableViewController.swift (hunk 1 above)
Comment (Camji55) [reply Loop/View Controllers/StatusTableViewController.swift]: Good question! Let me take a look! This was the only non-preview instance of DismissableHostingController that didn't have the proper color palette.

Comment (Camji55) [reply Loop/View Controllers/StatusTableViewController.swift]: Oh yeah, that's much better.

**Pull request merged**